querydsl-codegen/src/test/java/com/querydsl/codegen/GenericExporterTest.java

```diff
@@ -117,5 +117,11 @@ public void Export_Domain_Package() {
         exporter.export(Cat.class.getPackage());
     }
 
+    @Test
+    public void Export_SerializerConfig() {
+        exporter.setTargetFolder(new File("target/gen7"));
+        exporter.setSerializerConfig(new SimpleSerializerConfig(true, true, true, true, ""));
+        exporter.export(Cat.class.getPackage());
```

Comment:
If we don't assert anything then how would the test fail? I don't see any exceptions that would be triggered. I can work on easing the process if you think that would help.